modules/inmarBidAdapter.js

```diff
@@ -50,7 +50,7 @@ export const spec = {
       method: 'POST',
       url: 'https://prebid.owneriq.net:8443/bidder/pb/bid',
       options: {
-        withCredentials: false
+        withCredentials: true
```

Comment:
It's not necessary for bidders to set withCredentials to true because the bidder factory does it for you. 
https://github.com/prebid/Prebid.js/blob/47e96ad7ae62040ede8c9b84a192e0b718d9947a/src/adapters/bidderFactory.js#L266

I know it's a bit nitpicky since setting to true does do any harm but it's always better not to have necessary code

Reply:
@mike-chowla Ok.  This is what I did originally, but @smenzer told me to put it back.  I have now removed it again.

Do either of you know how to fix the CircleCI errors?  I thought I had ironed out these problems when I made the initial adapter submission a few months ago, but I'm seeing new errors outside the scope of my changes.  I'm guessing it's some kind of settings issue, but I don't know how to fix this.  Would greatly appreciate some help.

Reply:
good catch @mike-chowla I didn't know the bidder factory handled that...sorry for the back and forth @zandree-owneriq.

circleci is having some issues right now and the prebid team is looking into them, but nothing you need to do right now.